apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -149,6 +149,9 @@ export class BookingsController {
   ): Promise<ApiResponse<unknown>> {
     const oAuthClientId = clientId?.toString();
     const locationUrl = body.locationUrl;
+
+    const { orgSlug } = req.body;
```

Comment:
```
const {orgSlug, locationUrl } = body
```

Reply:
Implemented in [refactor: destructure properies from body instead of req.body](https://github.com/calcom/cal.com/pull/14525/commits/485c5218825af831da82bf76a035e22c443692f3)